src/Sulu/Component/PHPCR/SessionManager/SessionManagerInterface.php

```diff
@@ -34,6 +36,8 @@ public function getSession();
      * @param string $segment
      *
      * @return NodeInterface
+     *
+     * @deprecated Do not use anymore, because the node is always returned from the same session.
```

Comment:
What do you suggest? `from the default session, although multiple sessions can exist`?